api/src/pcapi/routes/backoffice/venues/blueprint.py

```diff
@@ -74,6 +75,18 @@ def _get_venues(form: forms.GetVenuesListForm) -> list[offerers_models.Venue]:
         sa.orm.joinedload(offerers_models.Venue.venueLabel).load_only(offerers_models.VenueLabel.label),
     )
 
+    if form.q.data:
+        search_query = form.q.data
+        terms = search_utils.split_terms(search_query)
+
+        if search_query.isnumeric():
+            base_query = base_query.filter(offerers_models.Venue.id == int(search_query))
+        elif all(term.isnumeric() for term in terms):
+            base_query = base_query.filter(offerers_models.Venue.id.in_([int(term) for term in terms]))
+        else:
+            name_query = "%{}%".format(search_query)
+            base_query = base_query.filter(offerers_models.Venue.name.ilike(name_query))
```

Comment:
Selon moi, il faudrait probablement, de la même manière que la recherche de lieu par ailleurs : 
- ignorer les accents dans la recherche
- chercher aussi dans le nom d'usage du lieu
```suggestion
            name_query = f"%{clean_accents(search_query)}%"
            base_query = base_query.filter(
                sa.or_(
                    sa.func.unaccent(offerers_models.Venue.name).ilike(name_query),
                    sa.func.unaccent(offerers_models.Venue.publicName).ilike(name_query),
                )
            )
```

Reply:
ok, question par curiosité, pourquoi d'un point de vue métier il y a un nom et un nom d'usage?

Reply:
Le nom est la raison sociale de l'établissement, le nom déclaré à l'INSEE, il n'est pas modifiable.
Le nom d'usage est un nom modifiable par l'AC, qui correspond à son nom commercial.
Par exemple un entrepreneur individuel a forcément son nom personnel comme nom juridique, mais peut vouloir présenter un joli nom d'activité.